### PR TITLE
Added a method to get the topic for the controller state callback.

### PR DESCRIPTION
**Pull request by ugocupcic**
Gazebo test failing. Need to fix it before merging.

### CODE REVIEW

**Comment by ugocupcic** [comment]
Tests is now working

File: sr_hand/test/hand_commander_test.cpp [hunk 1]
```diff
@@ -16,6 +16,14 @@
     EXPECT_TRUE(true);
 }
 
+TEST(HandCommander, topic_state)
```
Comment (markpitchless) [inline sr_hand/test/hand_commander_test.cpp:19]: Nice to see a test going in :)

Comment (ugocupcic) [inline sr_hand/test/hand_commander_test.cpp:19]: I knew it'd make you happy :o)

File: sr_hand/test/hand_commander_test.cpp [hunk 2]
```diff
@@ -16,6 +16,14 @@
...
+{
+    HandCommander handcmd = HandCommander();
+
+    std::string topic = handcmd.get_controller_state_topic("ffj3");
+    EXPECT_EQ("/sh_ffj3_mixed_position_velocity_controller/state", topic);
+}
```
Comment (markpitchless) [inline sr_hand/test/hand_commander_test.cpp:28]: Should test the negative state here as well, ie what happens when we pass an unknown joint name. Inspection suggests we get an empty string.

File: sr_hand/test/hand_commander_test.cpp [hunk 3]
```diff
@@ -40,5 +48,9 @@ int main(int argc, char **argv)
     testing::InitGoogleTest(&argc, argv);
     ros::init(argc, argv, "hand_commander_test");
     ros::NodeHandle nh; // init the node
+
+    //sleep until gazebo is ready
```
Comment (markpitchless) [inline sr_hand/test/hand_commander_test.cpp:55]: We could do with finding a better way to do this but it works for now.

File: sr_hand/test/hand_commander_test.cpp (hunk 2 above)
Comment (ugocupcic) [inline sr_hand/test/hand_commander_test.cpp:28]: Fixed in latest push. Can we merge now?

**Pull request merged**